dagster/implnets/templates/v1/implnet_ops_SOURCEVAL.py

```diff
@@ -565,13 +572,13 @@ def SOURCEVAL_graph_reports(context, msg: str):
 def harvest_SOURCEVAL():
     harvest = SOURCEVAL_gleaner()
 
-    report1 =SOURCEVAL_missingreport_s3(harvest)
-    #report1 = missingreport_s3(harvest, source="SOURCEVAL")
+    report1 = SOURCEVAL_missingreport_s3(harvest)
     load1 = SOURCEVAL_nabu_prune(harvest)
     load2 = SOURCEVAL_nabuprov(load1)
     load3 = SOURCEVAL_nabuorg(load2)
     load4 = SOURCEVAL_naburelease(load3)
     load5 = SOURCEVAL_uploadrelease(load4)
-    report2=SOURCEVAL_missingreport_graph(load5)
-    report3=SOURCEVAL_graph_reports(report2)
+    report2 = SOURCEVAL_missingreport_graph(load5)
+    report3 = SOURCEVAL_graph_reports(report2)
+    report4 = SOURCEVAL_identifier_stats(report3)
```

Comment:
think harvest, or report1 is fine here...  The identifier just hits the s3.

the two last processes that can hit the graph are done one after another.